src/pages/profile/advanced/model.ts

```diff
@@ -0,0 +1,52 @@
+import { AnyAction, Reducer } from 'redux';
+
+import { EffectsCommandMap } from 'dva';
+import { AdvancedProfileData } from './data.d';
+import { queryAdvancedProfile } from './service';
+
+export type Effect = (
+  action: AnyAction,
+  effects: EffectsCommandMap & { select: <T>(func: (state: AdvancedProfileData) => T) => T },
+) => void;
+
+export interface ModelType {
+  namespace: string;
+  state: AdvancedProfileData;
+  effects: {
+    fetchAdvanced: Effect;
+  };
+  reducers: {
+    show: Reducer<AdvancedProfileData>;
+  };
+}
+
+const Model: ModelType = {
+  namespace: 'profileAndadvanced',
+
+  state: {
+    advancedOperation1: [],
+    advancedOperation2: [],
+    advancedOperation3: [],
+  },
+
+  effects: {
+    *fetchAdvanced(_, { call, put }) {
+      const response = yield call(queryAdvancedProfile);
+      yield put({
+        type: 'show',
+        payload: response,
+      });
+    },
+  },
+
+  reducers: {
+    show(state, { payload }) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: '{ payload }' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)